controllers/questions.ts

```diff
@@ -0,0 +1,63 @@
+const crypto = require("crypto");
+import { Request, Response } from "express";
+
+import { Question } from "../types/questions";
+import { CustomRequest, CustomResponse } from "../types/global";
+
+const logger = require("../utils/logger.ts");
+const { HEADERS_FOR_SSE } = require("../constants/constants");
+const { INTERNAL_SERVER_ERROR } = require("../constants/errorMessages");
+
+const questionQuery = require("../models/questions");
+
+let clients = [];
```

Comment:
- Should we specify the type as this is a ts file? 
- Can you please explain what this `clients` array does?

Reply:
- Yes, we can do so.
- This array contains the list of connected clients for which the data will be send realtime.